src/test/util/setup_common.cpp

```diff
@@ -184,10 +185,7 @@ ChainTestingSetup::ChainTestingSetup(const std::string& chainName, const std::ve
     ::sporkManager = std::make_unique<CSporkManager>();
     ::governance = std::make_unique<CGovernanceManager>();
     ::masternodeSync = std::make_unique<CMasternodeSync>(*m_node.connman);
-    ::coinJoinServer = std::make_unique<CCoinJoinServer>(*m_node.mempool, *m_node.connman, *::masternodeSync);
-#ifdef ENABLE_WALLET
-    ::coinJoinClientQueueManager = std::make_unique<CCoinJoinClientQueueManager>(*m_node.connman, *::masternodeSync);
-#endif // ENABLE_WALLET
+    m_node.cj_ctx = std::make_unique<CJContext>(*m_node.connman, *m_node.mempool, *::masternodeSync, true);
```

Comment:
for better readability without IDE could you add here `/* fRelayConnection = */ or similar comment?

Reply:
Resolved in latest push